NEXT_CHANGELOG.md

```diff
@@ -32,6 +32,44 @@ For `experimental_cache` with redis caching it now works with only a single Redi
 
 By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/2310
 
+### Override the root certificate list for subgraph requests ([Issue #1503](https://github.com/apollographql/router/issues/1503))
```

Comment:
Once the behavior on replacing v.s. augmenting the root store is decided (see other comments), let’s document it explicitly.